keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -55,7 +48,7 @@ class ResourceActuator(
       val resource = resourceRepository.get(name, type)
 
       val (desired, current) = plugin.resolve(resource)
-      val diff = differ.compare(desired, current)
+      val diff = ResourceDiff(desired, current)
```

Comment:
Instead of using the diff library directly we just use `ResourceDiff` everywhere